app/models/jwt_token.rb

```diff
@@ -0,0 +1,63 @@
+require 'jwt'
+require 'base64'
+
+class JwtToken < Struct.new(:token)
+  class << self
+    def encode(user, secret)
+      payload = prepare_payload(user, secret)
+      new JWT.encode(payload, secret)
+    end
+
+    private
+
+    def prepare_payload(user, secret)
+      jti_raw = [secret, iat].join(':')
+      jti = Digest::MD5.hexdigest(jti_raw)
```

Comment:
why md5 and not sha2 for example?
If my memory serves me right, the default is sha2-256